src/main/dml/fenixedu-academic.dml

```diff
@@ -1356,6 +1356,44 @@ valueType org.fenixedu.academic.domain.util.i18n.Languages as Languages {
         LocalizedString name (REQUIRED);
     }
 
+    class student.personaldata.EducationLevelType {
+        String code (REQUIRED);
+        LocalizedString name (REQUIRED);
+        boolean active;
+        boolean forStudent;
+        boolean forStudentHousehold;
+        boolean forMobilityStudent;
+        boolean other;
+        boolean phDDegree;
+        boolean schoolLevelBasicCycle;
+        boolean highSchoolOrEquivalent;
+        boolean higherEducation;
+    }
```

Comment:
For now we can migrate all these slots. But when replacing the enum references, if we find some of these that aren't being used, we should also remove them from the entity.